security/pkg/stsservice/tokenmanager/google/tokenexchangeplugin.go

```diff
@@ -313,11 +365,15 @@ func (p *Plugin) generateSTSResp(atResp *accessTokenResponse) ([]byte, error) {
 	} else {
 		expireInSec = int64(time.Until(exp).Seconds())
 	}
+	return p.generateSTSRespInner(atResp.AccessToken, expireInSec)
+}
+
+func (p *Plugin) generateSTSRespInner(token string, expire int64) ([]byte, error) {
 	stsRespParam := stsservice.StsResponseParameters{
-		AccessToken:     atResp.AccessToken,
+		AccessToken:     token,
 		IssuedTokenType: tokenType,
 		TokenType:       "Bearer",
-		ExpiresIn:       expireInSec,
+		ExpiresIn:       expire,
 	}
 	statusJSON, err := json.MarshalIndent(stsRespParam, "", " ")
 	if pluginLog.DebugEnabled() {
```

Comment:
"redacted"?

Reply:
If expire time is redacted, then the debug mode does not provide useful information.